### PR TITLE
equations working fine with vuepress-latex-plugin

### PR DESCRIPTION
**Pull request by jurra**
Make sure to do `npm install` before testing to update the packages installed locally.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,2 @@
-node_modules
-
-# vuepress build output
-docs/.vuepress/dist
```
Comment (martinjaeger) [inline .gitignore]: Why did you remove this? I think it makes sense to keep it because we don't want to publish the vuepress build output.

**Review by martinjaeger — CHANGES_REQUESTED** [review]
Just a question regarding .gitignore changes. Otherwise thanks for finding a solution for the math equations!

File: .gitignore (hunk 1 above)
Comment (jurra) [reply .gitignore]: because those files are just need it to build the site, they are not part of the source code. If you commit the `node_modules` you would be commiting actually all those dependencies. The practice is just to keep the reference to them in the `package.json`.

Comment (martinjaeger) [reply .gitignore]: Yes, I know. That's why `node_modules` was already present in .gitignore before. But your commit removed the line `docs/.vuepress/dist` from .gitignore. I think it should be kept, same as `yarn.lock`.

Comment (jurra) [reply .gitignore]: Ok this is now corrected.

**Review by martinjaeger — APPROVED** [review]
